static/index.js

```diff
@@ -79,6 +79,14 @@ function updateNavigation() {
     }
 
     var navbarEle = document.getElementById("navbar")
+    if (navbarEle === null) {  // create it
+        navbarEle = document.createElement("div");
+        navbarEle.id="navbar";
+        navOuterEle = document.createElement("nav")
+        navOuterEle.appendChild(navbarEle);
+        document.body.prepend(navOuterEle);
```

Comment:
it seems like javascript is a surprisingly verbose language when it comes to trying to define HTML elements.
This is the code to make `<nav><div id="navbar"></div></nav>`.
I am probably missing some better way to do it.

Reply:
Ugh. A comment might be useful, but it's fine without it too.